.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ build-and-test-commands: &build-and-test-commands
     find . -type f -regex ".*/target/failsafe-reports/.*xml" -exec cp {} ~/project/artifacts/junit/ \;
 
 release-args: &release-args
-  mvn-release-perform-command: mvn --batch-mode release:perform -s .circleci/.maven.xml -PbuildKar
+  mvn-release-perform-command: mvn --batch-mode release:perform -s .circleci/.maven.xml -PbuildKar,circle-ci-release
```

Comment:
Maybe the pom can be changed such that only our "hidden" internal Jenkins CI build needs a specific profile? 

If not, please be certain you also change the `nexus-format-archetype` to include this and related changes.